src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -19,10 +20,13 @@ public class AddTaskCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to the address book. "
             + "Parameters: "
             + PREFIX_TASK_TITLE + "TITLE "
-            + PREFIX_TASK_NOTE + "NOTE\n"
+            + PREFIX_TASK_NOTE + "NOTE"
+            + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_TASK_TITLE + "Get flowers "
-            + PREFIX_TASK_NOTE + "Wedding Anniversary";
+            + PREFIX_TASK_NOTE + "Wedding Anniversary"
```

Comment:
Could you include a whitespace at the end?
```suggestion
            + PREFIX_TASK_NOTE + "Wedding Anniversary "
```
<img width="668" alt="Screenshot 2023-10-25 at 11 03 37 AM" src="https://github.com/AY2324S1-CS2103T-T10-2/tp/assets/79037826/6ab2500f-8681-434d-a246-35c24369f466">